luaui/actions.lua

```diff
@@ -262,7 +243,7 @@ function actionHandler:TextAction(line)
   end
   -- remove the command from the words list and the raw line
   table.remove(words, 1)
-  local _,_,line = string.find(line, "[^%s]+[%s]+(.*)")
+  _,_,line = string.find(line, "[^%s]+[%s]+(.*)")
```

Comment:
Why is line global here?

Reply:
It isn't, it's the local var from line 238.

Reply:
_ is reserved keyword

We are fixing a redundant `local` variable assignment

Reply:
Thanks for clearing that up!